layouts/community/60_ansi/stanrc85-ansi/rules.mk

```diff
@@ -0,0 +1,2 @@
+USER_NAME := stanrc85-ansi
```

Comment:
Unless you're using a userspace (and it doesn't appear that you are), this is not necessary and should be removed.

```suggestion
```

Reply:
Changes made.